src/tests/ftest/cart/corpc/cart_corpc_five_node.yaml

```diff
@@ -24,7 +24,7 @@ hosts: !mux
 tests: !mux
   corpc_prefwd:
     name: corpc_prefwd
-    srv_bin: ../bin/crt_launch
-    srv_arg: "-e tests/test_corpc_prefwd"
-    srv_env: ""
-    srv_ppn: "5"
+    test_servers_bin: ../../../../bin/crt_launch
```

Comment:
do we need to specify relative path? i think launch.py may have included daos/install/bin in PATH. can you check?

Reply:
To be fixed in [DAOS-5566](https://jira.hpdd.intel.com/browse/DAOS-5566) (_Remove relative paths from CaRT test YAML_)